webpack/assets/javascripts/react_app/components/Bookmarks/index.js

```diff
@@ -13,9 +15,14 @@ const mapStateToProps = (state, { controller }) => ({
   errors: selectBookmarksErrors(state, controller),
   bookmarks: selectBookmarksResults(state, controller),
   status: selectBookmarksStatus(state, controller),
-  showModal: selectBookmarksShowModal(state),
+  isModalOpen: selectIsModalOpen(state, BOOKMARKS_MODAL),
 });
 
+const mapDispatchToProps = {
+  ...bookmarksActions,
+  ...bindForemanModalActionsToId({ id: BOOKMARKS_MODAL }), // gives us setModalOpen and setModalClosed
```

Comment:
See my other comment, but also I wanted there to be an example in Foreman of using ForemanModal with `connect` etc.

Reply:
As we decided to go with the hooks conventions, in the near future we won't be using `connect` at all, my fear is that people will copy it. it can be in a follow-up PR too.